### ISSUE TITLE
Nevus psiloliparus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9636]
**Preferred term label:**
Nevus psiloliparus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Nevus psiloliparus is a rare fatty tissue nevus that is a marker for encephalocraniocutaneous lipomatosis, a neurocutaneous syndrome with ocular and central nervous system anomalies. Clinically, nevus psiloliparus is often described as a congenital alopecia and appears as an irregularly shaped, circumscribed area of alopecia on the scalp. Histopathology demonstrates a near-complete absence of mature hair follicles with preservation of arrector pili muscles and mature adipocytes within the dermis.PMID: 32034785

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613001 
PMID: 26942290   5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
